styles.css

```diff
@@ -176,6 +176,336 @@ h2 {
   padding-bottom: 10px;
 }
 
+.hide_mobile {
+  display: none;
+}
+
+.show_mobile {
+  display: initial;
+}
+
+#popwinmobile {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: #fff;
+  z-index: 1;
+  padding-bottom: 10%;
+}
+
+.popx img {
+  margin-top: 8%;
+  margin-right: 5%;
+  margin-bottom: 0%;
+  padding-left: 85%;
+}
+
+.poph2 {
+  margin-left: 5%;
+  margin-top: 1%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 20px;
+  line-height: 80%;
+  color: #1a2236;
+  margin-bottom: 2%;
+}
+
+.projects1b3pop {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0;
+}
+
+.b3pop {
+  display: flex;
+  padding: 0%;
+  margin-left: 6%;
+}
+
+.sbuttonpop {
+  row-gap: 1%;
+  width: fit-content;
+  height: 25px;
+  color: #1a2236;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  margin-right: 15px;
+  font-size: 10.3334px;
+  font-weight: 300;
+  text-align: center;
+  border: 1px solid;
+}
+
+.popmain {
+  margin-left: 5%;
+  margin-right: 5%;
+  margin-top: 5%;
+}
+
+.ima4pop {
+  display: flex;
+  padding: 0%;
+  margin-top: 0%;
+  margin-left: 5%;
+  margin-right: 5%;
+  column-gap: 0%;
+}
```

Comment:
- [ ] [OPTIONAL] Great job making the changes requested by the previous reviewer 💯, but I think it could be better if you fix the width of the small images, it should be aligned to the main image ⚡,  your project will look similar to the Figma template.
- [ ] [OPTIONAL] Please, also check the same section in the mobile version. 🔢

![image](https://user-images.githubusercontent.com/91301423/202586430-f526540f-9bbc-4140-b6ff-711e5741174a.png)